client/packages/glsp-sprotty/src/features/tool-manager/tool-manager.ts

```diff
@@ -152,11 +153,18 @@ export class ToolManagerActionHandlerInitializer implements IActionHandlerInitia
 }
 
 @injectable()
-export class StandardToolsEnablingKeyListener extends KeyListener {
+export class StandardToolsEnablingKeyListener extends ExtendedKeyListener {
     keyDown(element: SModelElement, event: KeyboardEvent): Action[] {
         if (matchesKeystroke(event, 'Escape')) {
             return [new EnableStandardToolsAction()];
         }
         return [];
     }
+
+    keyUp(element: SModelElement, event: KeyboardEvent): Action[] {
+        if (matchesKeystroke(event, 'ControlLeft' || 'ControlRight')) {
```

Comment:
I think I'm not really in favor of this fix for the following reasons:

- The StandardToolsEnablingKeyListener is a very general thing -- ie enabling hitting escape to exit any special mode whatsoever. By adding this Control key handling, it becomes something that handles a very specific issue in the stamp mode.
- It seems that this may introduce very unforeseen side-effects. E.g. selecting a tool and hitting Ctrl will disable the tool. Hitting Ctrl once should usually not have any side-effects.
- It breaks the normal behavior of the stamp mode: Ctrl-down click Ctrl-up Ctrl-down click -- with this listener it only works if you keep holding down the Ctrl key.